include/z64transition.h

```diff
@@ -49,8 +49,19 @@ typedef struct {
     /* 0x254 */ char unk_254[0x4];
 } TransitionContext; // size = 0x258
 
+typedef enum {
+    /* 1 */ TRANS_INSTANCE_TYPE_END = 1,
+    /* 2 */ TRANS_INSTANCE_TYPE_START,
+} TransitionInstanceType;
+
+#define TRANS_INSTANCE_TYPE_FADE_FLASH 3
```

Comment:
No, it is meant to be separate. Based on https://github.com/Decompollaborate/mm/blob/play/src/code/z_play.c#L671-L672, 1 and 2 is shared between all types of instance transitions. On the other hand, `TRANS_INSTANCE_TYPE_FADE_FLASH` is meant just for `Fade`, so I didn't think it would be appropriate for it to be apart of the same enum.